src/modules/dom.js

```diff
@@ -0,0 +1,30 @@
+const restorauntObjects = 'https://www.themealdb.com/api/json/v1/1/categories.php';
+const main = document.querySelector('main');
+
+// data.categories[0].idCategory
+// console.log(data.categories[0].strCategory)
+// data.categories.forEach((elem) => displayData(elem))
```

Comment:
This comment could also be removed to make sure we keep our code well tidied up. Just a suggestion though:smile: